app/code/Magento/ImportExport/Block/Adminhtml/Export/Filter.php

```diff
@@ -41,17 +42,35 @@ class Filter extends \Magento\Backend\Block\Widget\Grid\Extended
     ];
 
     /**
+     * @var \Magento\ImportExport\Model\Export\ConfigInterface
+     */
+    private $exportConfig;
+
+    /**
+     * @var \Magento\CatalogImportExport\Model\Export\Product\Type\Factory
+     */
+    private  $typeFactory;
+
+    /**
+     * Filter constructor.
+     * @param \Magento\ImportExport\Model\Export\ConfigInterface $exportConfig
+     * @param \Magento\CatalogImportExport\Model\Export\Product\Type\Factory $typeFactory
      * @param \Magento\Backend\Block\Template\Context $context
      * @param \Magento\Backend\Helper\Data $backendHelper
      * @param \Magento\ImportExport\Helper\Data $importExportData
      * @param array $data
      */
+
     public function __construct(
+        \Magento\ImportExport\Model\Export\ConfigInterface $exportConfig = null,
+        \Magento\CatalogImportExport\Model\Export\Product\Type\Factory $typeFactory = null,
         \Magento\Backend\Block\Template\Context $context,
         \Magento\Backend\Helper\Data $backendHelper,
         \Magento\ImportExport\Helper\Data $importExportData,
         array $data = []
     ) {
+        $this->exportConfig = $exportConfig ?: \Magento\Framework\App\ObjectManager::getInstance()->get(\Magento\ImportExport\Model\Export\ConfigInterface::class);
```

Comment:
I believe these lines are exceeding the 120 character limit, could you please reformat them not to exceed

Reply:
sure